tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -432,5 +540,21 @@ public static void Constructors_VerifyGuardClauses()
 
             response.Result.Should().BeOfType<NoContentResult>();
         }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task UpdateStatusAsync_OrderIsComplete_InvokesGetOrderCompletedStatus(
+            [Frozen] Mock<IOrderService> service,
+            [Frozen] CallOffId callOffId,
+            Order order,
+            StatusModel model,
+            OrdersController controller)
+        {
+            service.Setup(o => o.GetOrderCompletedStatus(It.IsAny<CallOffId>())).ReturnsAsync(order);
+
+            var response = await controller.UpdateStatusAsync(callOffId, model);
```

Comment:
Looks like we're not using `response` here so we can remove it.

Reply:
Removed variable `response`